bundles/org.openhab.binding.evcc/src/main/java/org/openhab/binding/evcc/internal/api/dto/Result.java

```diff
@@ -160,16 +163,23 @@ public String getBatteryMode() {
     }
 
     /**
-     * @return whether grid is configured
+     * @return grid's currents
+     */
+    public float[] getGridCurrents() {
+        return gridCurrents;
+    }
+
+    /**
+     * @return grid's energy
      */
-    public boolean getGridConfigured() {
-        return gridConfigured;
+    public float getGridEnergy() {
+        return gridEnergy;
     }
 
     /**
      * @return grid's power
      */
-    public float getGridPower() {
+    public Float getGridPower() {
```

Comment:
Please mention in the JavaDoc that this can be null.

Reply:
done